src/routes/reminder/+server.ts

```diff
@@ -1,36 +1,36 @@
 import { PrismaClient } from '@prisma/client';
 import { json } from '@sveltejs/kit';
-import * as cron from 'node-cron';
 
 const prisma = new PrismaClient();
 
-export function POST({ fetch }: { fetch: any }) {
+export async function POST({ fetch }: { fetch: any }) {
 	/**
         goes through each user in the db
         If it's time to send them a notif, we'll just do so right there. No need to schedule it.
     */
-	async function sendNotif() {
```

Comment:
The reminder route should just invoke the lib/server/reminders code, where all this logic is moved to